control.go

```diff
@@ -130,6 +130,7 @@ func (c *controlConn) shuffleDial(endpoints []string) (conn *Conn, err error) {
 		}
 
 		hostInfo, _ := c.session.ring.addHostIfMissing(host)
+		// TODO (gus) use translated address here?
```

Comment:
I dont think we need to, as `session.connect` will translate the address for us below